packages/react-ui-kit/src/Identity/AvatarGrid.tsx

```diff
@@ -46,24 +47,25 @@ const avatarGridStyle: <T>(props: Props<T>) => ObjectInterpolation<undefined> =
     display: 'grid',
     gridGap: borderWidth,
     gridTemplateColumns: 'repeat(2, 1fr)',
+    height: `${size}px`,
     justifyItems: 'center',
-    minHeight: `${size + borderWidth * 2}px`,
     overflow: 'hidden',
+    width: `${size}px`,
   };
 };
 
 const filteredAvatarGridProps = (props: Props) =>
-  filterProps(props, ['backgroundColor', 'borderColor', 'items', 'size']);
+  filterProps(props, ['backgroundColor', 'borderColor', 'items', 'size', 'borderWidth']);
 
-export const AvatarGrid = (props: Props) => {
-  const {size, items} = props;
+export const AvatarGrid = ({borderWidth = 1, size = DEFAULT_AVATAR_SIZE + borderWidth * 2, items, ...props}: Props) => {
```

Comment:
I think to be perfect.. we need to subtract the borderWidth * 2
```suggestion
export const AvatarGrid = ({borderWidth = 1, size = DEFAULT_AVATAR_SIZE - borderWidth * 2, items, ...props}: Props) => {
```